requirements.txt

```diff
@@ -1,6 +1,7 @@
 click
 astropy
 networkx
+numpy >= 1.17, <2
```

Comment:
If middleware is Breaking with numpy 2 can you file a ticket please and include the error message?

Reply:
For reference, [this](https://rubinobs.atlassian.net/browse/DM-44915) is the ticket I filed.